src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/SecurityServersApiController.java

```diff
@@ -71,4 +82,20 @@ public SecurityServersApiController(GlobalConfService globalConfService,
         SecurityServer securityServer = securityServerConverter.convert(securityServerId);
         return new ResponseEntity<>(securityServer, HttpStatus.OK);
     }
+
+    @Override
+    @PreAuthorize("hasAuthority('VIEW_SECURITY_SERVERS')")
```

Comment:
Pending question about permissions needs clarification https://niis.slack.com/archives/GGBHVCYSY/p1579849271003200

Reply:
This has been clarified to be fine